ui-v2/src/components/flow-runs/data-table/tasks-cell.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+	type FlowRunWithDeploymentAndFlow,
+	type FlowRunWithFlow,
+} from "@/api/flow-runs";
+import { buildGetFlowRusTaskRunsCountQuery } from "@/api/task-runs";
+import { Icon } from "@/components/ui/icons";
+
+import { Typography } from "@/components/ui/typography";
+import { pluralize } from "@/utils";
+import { useSuspenseQuery } from "@tanstack/react-query";
+import clsx from "clsx";
+
+type TasksCellsProp = {
+	flowRun: FlowRunWithFlow | FlowRunWithDeploymentAndFlow;
+};
+
+export const TasksCell = ({ flowRun }: TasksCellsProp) => {
+	const { data } = useSuspenseQuery(
+		buildGetFlowRusTaskRunsCountQuery([flowRun.id]),
+	);
+
+	const taskRunsCount = data[flowRun.id];
+
+	if (taskRunsCount === undefined) {
+		return "Tasks not found";
+	}
```

Comment:
I'd probably just return `null` here not not show anything if there are no task runs. Also might be worth checking with adam if we even want to have this column. Since its going to create an api call for each row.